provision/infinispan/ispn-helm/templates/infinispan.yaml

```diff
@@ -1,4 +1,13 @@
 {{- $namespace := .Values.namespace | required ".Values.namespace is required." -}}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: connect-secret
```

Comment:
@kami619 the secret is generated here with the password from AWS.

Reply:
great, thanks

Reply:
in this case we don't need it to be a variable.